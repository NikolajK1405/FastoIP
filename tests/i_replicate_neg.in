fun bool writeBool(bool b) = write(b)

fun [bool] main() =
  let fs = replicate(true, -2) in
  map(writeBool, fs)
