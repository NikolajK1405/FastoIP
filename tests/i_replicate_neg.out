Interpreter error: Bad type for argument of "replicate": expected int, but got bool (true) at line 4, column 12